### DISCUSSION TITLE
First tap builds and tests, but how to publish formula and bottle ?

### DISCUSSION THREAD

**Discussion by a4z** [Homebrew/discussions#4472] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
not applicable, GitHub actions / CI question
```

### Output of `brew doctor`

```shell
not applicable, GitHub actions / CI question
```

### Description of issue

I have my first Formula merged into the main branch of a repo. All Github actions were running successfully, 

My question is, how do the Formula and bottle get published so the default generated text of the readme works?
The bottle is a simple one, with no arch or platform specifics since I thought I started with the most simple use case.

the repo is here, but once I know better how that all works I plan to move it
https://github.com/a4z/homebrew-cpp-taps

**Comment by ZhongRuoyu** [comment, marked as answer]
To do so, you'll need to [add a label](https://github.com/a4z/homebrew-cpp-taps/labels) named `pr-pull`, and add that label to the PR ready to merge. A workflow will be triggered and the PR will be merged automatically, so you don't need to merge the PR yourself.

Taken from `brew tap-new` output:

```
When a pull request making changes to a formula (or formulae) becomes green
(all checks passed), then you can publish the built bottles.
To do so, label your PR as `pr-pull` and the workflow will be triggered.
```

**Reply by a4z** [reply]
Yes, thank you!
